source/installguide/hypervisor/kvm.rst

```diff
@@ -289,15 +289,7 @@ cloudstack-agent and should already be installed.
 #. Turning on "listen\_tcp" in libvirtd.conf is not enough, we have to
    change the parameters as well:
 
-   On RHEL or CentOS modify ``/etc/sysconfig/libvirtd``:
-
-   Uncomment the following line:
-
-   .. parsed-literal::
-
-      #LIBVIRTD_ARGS="--listen"
-
-   On Ubuntu 14.04: modify ``/etc/default/libvirt-bin``
+      On Ubuntu 14.04: modify ``/etc/default/libvirt-bin``
```

Comment:
It would also be good to remove Ubuntu 14.04 from documentation as it is no longer supported.